sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs

```diff
@@ -11,22 +11,22 @@ namespace Azure.Core.GeoJson
     public sealed class GeoBoundingBox : IEquatable<GeoBoundingBox>
     {
         /// <summary>
-        /// The westmost value of <see cref="GeoObject"/> coordinates.
+        /// The west-most value of <see cref="GeoObject"/> coordinates.
```

Comment:
Is the hyphen correct here - https://www.merriam-webster.com/dictionary/westmost

Reply:
```suggestion
        /// The westmost value of <see cref="GeoObject"/> coordinates.
```